docs/ops/normalization/BatchNormInference_1.md

```diff
@@ -4,39 +4,33 @@
+**Short description**: *BatchNormInference* performs Batch Normalization operation described in the [Batch Normalization: Accelerating Deep Network Training by Reducing Internal Covariate Shift](https://arxiv.org/abs/1502.03167v2) article.
 
-**Attributes**:
+**Detailed Description**
 
-* *epsilon*
-  * **Description**: *epsilon* is the number to be added to the variance to avoid division by zero when normalizing a value. For example, *epsilon* equal to 0.001 means that 0.001 is added to the variance.
-  * **Range of values**: a positive floating-point number
-  * **Type**: `float`
-  * **Default value**: None
-  * **Required**: *yes*
+*BatchNormInference* performs the following operations on a given data batch input tensor `data`:
 
-**Inputs**
+* Normalize each activation \f$x^{(k)}\f$ by means of the mean and variance.
+\f[
+   \hat{x}^{(k)}=\frac{x^{(k)} - E[x^{(k)}]}{\sqrt{Var(x^{(k)}) + \epsilon}}
+\f]
+where \f$E[x^{(k)}]\f$ and \f$Var(x^{(k)})\f$ are the mean and variance, calculated per the channel axis of `data` input, and correspond to `mean` and `variance` inputs respectively. Additionally, \f$\epsilon\f$ is a value added to the variance for numerical stability and corresponds to `epsilon` attribute.
 
-* **1**: `input` - input tensor with data for normalization. At least a 2D tensor of type T, the second dimension represents the channel axis and must have a span of at least 1. **Required.**
-* **2**: `gamma` - gamma scaling for normalized value. A 1D tensor of type T with the same span as input's channel axis. **Required.**
-* **3**: `beta` - bias added to the scaled normalized value. A 1D tensor of type T with the same span as input's channel axis.. **Required.**
-* **4**: `mean` - value for mean normalization. A 1D tensor of type T with the same span as input's channel axis.. **Required.**
-* **5**: `variance` - value for variance normalization. A 1D tensor of type T with the same span as input's channel axis.. **Required.**
+* Linear transformation of each normalized activation based on `gamma` and `beta` input, representing the scaling factor and shift respectively.
+\f[
+   \hat{y}^{(k)}=\gamma^{(k)}\hat{x}^{(k)} + \beta^{(k)}
+\f]
+where \f$\gamma^{(k)}\f$ and \f$\beta^{(k)}\f$ are learnable parameters, calculated along the channel axis, and correspond to `gamma` and `beta` inputs.
 
-**Outputs**
-
-* **1**: The result of normalization. A tensor of the same type and shape with 1st input tensor.
-
-**Types**
+**Mathematical Formulation**
 
-* *T*: any numeric type.
+Let `x` be a *d*-dimensional input, \f$x=(x_{1}\dotsc x_{d})\f$. Since normalization is applied to each activation \f$E[x^{(k)}]\f$, we may focus on a particular activation and ommit k.
 
-**Mathematical Formulation**
+Considering a mini-batch \f$\mathcal{B}\f$ of m values for a particular activation. *BatchNormInference* performs Batch Normalization algorithm, as follows:
```

Comment:
Considering a mini-batch \f$\mathcal{B}\f$ of m values for a particular activation. *BatchNormInference* performs Batch Normalization algorithm, as follows: --->
For a particular activation, consider a mini-batch \f$\mathcal{B}\f$ of m values. *BatchNormInference* performs Batch Normalization algorithm as follows:

Reply:
Done